helix-term/src/keymap.rs

```diff
@@ -2,6 +2,7 @@ pub mod default;
 pub mod macros;
 
 pub use crate::commands::MappableCommand;
+#[allow(unused_imports)] // `Config` is used in a macro
 use crate::config::Config;
```

Comment:
It ends up being used in the `tests` module below so you can remove the `use` here and add it to the `tests` module